scripts/collect-info.yaml

```diff
@@ -24,7 +28,7 @@
       - name: errors_info
         cmd: journalctl -p err --no-pager
       - name: etcd_info
-        cmd: etcdctl --peers={{ etcd_access_addresses }} cluster-health
+        cmd: etcdctl --peers={{ etcd_access_addresses | default("http://127.0.0.1:2379") }} cluster-health
```

Comment:
it's the default endpoint address in a generic sense. It doesn't apply to kargo deployments, but this makes it applicable getting logs from non-kargo deployments